.github/workflows/pylint.yml

```diff
@@ -18,7 +18,11 @@ jobs:
         python -m pip install --upgrade pip
         pip install pylint sphinx sphinx_rtd_theme
         pip install -r requirements.txt
-    - name: Test with pytest
+        pip install pytest
+    - name: Run pylint
       run: |
-        pylint --disable=I --disable=W --disable=C --disable=R `find . -type d \( -path ./p5/core/tests -o -path ./profiling -o -path ./p5/sketch/tests -o -path ./p5/pmath/tests \) \
-        -prune -false -o -name '*.py'`
+        pylint --disable=I --disable=W --disable=C --disable=R `find .  -path ./profiling -prune -false -o -name '*.py'`
+    - name: Run Tests
+      run: |
+        sudo apt-get install python-opengl libglfw3 -y
+        pytest -svv p5
```

Comment:
What does the -svv tag do?

Reply:
These tags just modify the output.
*pytest -v .* gives this (-v verbose )
```
p5/core/tests/test_color.py::TestColor::test_parser PASSED               [  2%]
p5/core/tests/test_geometry.py::TestBoxGeometry::test_normals PASSED     [  5%]
p5/core/tests/test_geometry.py::TestBoxGeometry::test_triangle_edges PASSED [  8%]
p5/core/tests/test_geometry.py::TestBoxGeometry::test_vertices PASSED    [ 10%]
p5/core/tests/test_shape.py::TestPShape::test_properties PASSED          [ 13%]
p5/core/tests/test_shape.py::TestPShape::test_transforms PASSED          [ 16%]
p5/core/tests/test_svg.py::TestSVG::test_get_style PASSED                [ 18%]
p5/core/tests/test_svg.py::TestSVG::test_parse_line PASSED               [ 21%]
p5/core/tests/test_svg.py::TestSVG::test_parse_rect PASSED               [ 24%]
p5/pmath/tests/test_curves.py::TestCurves::test_bezier_point PASSED      [ 27%]
p5/pmath/tests/test_curves.py::TestCurves::test_bezier_tangent PASSED    [ 29%]
p5/pmath/tests/test_curves.py::TestCurves::test_curve_point PASSED       [ 32%]
p5/pmath/tests/test_curves.py::TestCurves::test_curve_tangent PASSED     [ 35%]
p5/pmath/tests/test_curves.py::TestCurves::test_quadratic_point PASSED   [ 37%]
p5/pmath/tests/test_matrix.py::TestMatrix::test_look_at PASSED           [ 40%]
p5/pmath/tests/test_matrix.py::TestMatrix::test_magnitude PASSED         [ 43%]
p5/pmath/tests/test_matrix.py::TestMatrix::test_normalize PASSED         [ 45%]
p5/pmath/tests/test_matrix.py::TestMatrix::test_perspective_matrix PASSED [ 48%]
p5/pmath/tests/test_matrix.py::TestMatrix::test_rotation_matrix PASSED   [ 51%]
p5/pmath/tests/test_matrix.py::TestMatrix::test_scale_transform PASSED   [ 54%]
p5/pmath/tests/test_matrix.py::TestMatrix::test_translation_matrix PASSED [ 56%]
p5/pmath/tests/test_matrix.py::TestMatrix::test_triple_axis_rotation_matrix PASSED [ 59%]
p5/pmath/tests/test_utils.py::TestUtils::test_constrain PASSED           [ 62%]
p5/pmath/tests/test_utils.py::TestUtils::test_distance PASSED            [ 64%]
p5/pmath/tests/test_utils.py::TestUtils::test_lerp PASSED                [ 67%]
p5/pmath/tests/test_utils.py::TestUtils::test_magnitude PASSED           [ 70%]
p5/pmath/tests/test_utils.py::TestUtils::test_normalize PASSED           [ 72%]
p5/pmath/tests/test_utils.py::TestUtils::test_remap PASSED               [ 75%]
p5/pmath/tests/test_utils.py::TestUtils::test_sq PASSED                  [ 78%]
p5/pmath/tests/test_vector.py::TestVector::test_angle PASSED             [ 81%]
p5/pmath/tests/test_vector.py::TestVector::test_angle_between PASSED     [ 83%]
p5/pmath/tests/test_vector.py::TestVector::test_arithmetic PASSED        [ 86%]
p5/pmath/tests/test_vector.py::TestVector::test_cross PASSED             [ 89%]
p5/pmath/tests/test_vector.py::TestVector::test_distance PASSED          [ 91%]
p5/pmath/tests/test_vector.py::TestVector::test_dot PASSED               [ 94%]
p5/pmath/tests/test_vector.py::TestVector::test_lerp PASSED              [ 97%]
p5/pmath/tests/test_vector.py::TestVector::test_properties PASSED        [100%]
```
**pytest -sv .* give ( -s disables per-test capturing or print statements inside the code are not printed unless a test fails, increases visibility )
```
p5/core/tests/test_color.py::TestColor::test_parser PASSED
p5/core/tests/test_geometry.py::TestBoxGeometry::test_normals PASSED
p5/core/tests/test_geometry.py::TestBoxGeometry::test_triangle_edges PASSED
p5/core/tests/test_geometry.py::TestBoxGeometry::test_vertices PASSED
p5/core/tests/test_shape.py::TestPShape::test_properties PASSED
p5/core/tests/test_shape.py::TestPShape::test_transforms PASSED
p5/core/tests/test_svg.py::TestSVG::test_get_style PASSED
p5/core/tests/test_svg.py::TestSVG::test_parse_line PASSED
p5/core/tests/test_svg.py::TestSVG::test_parse_rect PASSED
p5/pmath/tests/test_curves.py::TestCurves::test_bezier_point PASSED
p5/pmath/tests/test_curves.py::TestCurves::test_bezier_tangent PASSED
p5/pmath/tests/test_curves.py::TestCurves::test_curve_point PASSED
p5/pmath/tests/test_curves.py::TestCurves::test_curve_tangent PASSED
p5/pmath/tests/test_curves.py::TestCurves::test_quadratic_point PASSED
p5/pmath/tests/test_matrix.py::TestMatrix::test_look_at PASSED
p5/pmath/tests/test_matrix.py::TestMatrix::test_magnitude PASSED
p5/pmath/tests/test_matrix.py::TestMatrix::test_normalize PASSED
p5/pmath/tests/test_matrix.py::TestMatrix::test_perspective_matrix PASSED
p5/pmath/tests/test_matrix.py::TestMatrix::test_rotation_matrix PASSED
p5/pmath/tests/test_matrix.py::TestMatrix::test_scale_transform PASSED
p5/pmath/tests/test_matrix.py::TestMatrix::test_translation_matrix PASSED
p5/pmath/tests/test_matrix.py::TestMatrix::test_triple_axis_rotation_matrix PASSED
p5/pmath/tests/test_utils.py::TestUtils::test_constrain PASSED
p5/pmath/tests/test_utils.py::TestUtils::test_distance PASSED
p5/pmath/tests/test_utils.py::TestUtils::test_lerp PASSED
p5/pmath/tests/test_utils.py::TestUtils::test_magnitude PASSED
p5/pmath/tests/test_utils.py::TestUtils::test_normalize PASSED
p5/pmath/tests/test_utils.py::TestUtils::test_remap PASSED
p5/pmath/tests/test_utils.py::TestUtils::test_sq PASSED
p5/pmath/tests/test_vector.py::TestVector::test_angle PASSED
p5/pmath/tests/test_vector.py::TestVector::test_angle_between PASSED
p5/pmath/tests/test_vector.py::TestVector::test_arithmetic PASSED
p5/pmath/tests/test_vector.py::TestVector::test_cross PASSED
p5/pmath/tests/test_vector.py::TestVector::test_distance PASSED
p5/pmath/tests/test_vector.py::TestVector::test_dot PASSED
p5/pmath/tests/test_vector.py::TestVector::test_lerp PASSED
p5/pmath/tests/test_vector.py::TestVector::test_properties PASSED

```